### PR TITLE
[Teletext] Fix flashing regression due to std:chrono change

### PR DESCRIPTION
**Pull request by enen92**
## Description
This fixes another regression from the move to `std::chrono` which leads for the flashing operation in Teletext to happen at a much higher rate / frequency than the one expected. Results were compared to an actual tv displaying the same page (in this case it flashes at 1Hz - 1 time per second).

See explanation in https://stackoverflow.com/questions/21050994/stdchrono-default-duration-for-time-since-epoch

This might explain why https://github.com/xbmc/xbmc/issues/21317 also got broken...

**Master:**

https://www.youtube.com/watch?v=oWqP27r2s4M

**PR:**

https://www.youtube.com/watch?v=6yS8jwJXoRA

### CODE REVIEW

File: xbmc/video/Teletext.cpp [hunk 1]
```diff
@@ -1337,7 +1337,10 @@ void CTeletextDecoder::DoFlashing(int startrow)
   /* Flashing */
   TextPageAttr_t flashattr;
   char flashchar;
-  long flashphase = std::chrono::steady_clock::now().time_since_epoch().count() % 1000;
+  long flashphase = std::chrono::duration_cast<std::chrono::milliseconds>(
```
Comment (lrusak) [inline xbmc/video/Teletext.cpp:1340]: Could you change flashphase to be `std::chrono::milliseconds`. Then you can drop the `.count()` and use `% 1000ms`.

That way we can update the existing comparisons with the `ms` literal. That way we know we are comparing milliseconds.

Comment (enen92) [reply xbmc/video/Teletext.cpp]: done

File: xbmc/video/Teletext.cpp [hunk 2]
```diff
@@ -1369,35 +1373,55 @@ void CTeletextDecoder::DoFlashing(int startrow)
...
+                if (flashphase < 250ms)
+                  doflash = true;
+                break;
+              case 2:
+                if (flashphase >= 250ms && flashphase < 500ms)
+                  doflash = true;
+                break;
+              case 3:
+                if (flashphase >= 500ms && flashphase < 750ms)
+                  doflash = true;
```
Comment (ksooo) [inline xbmc/video/Teletext.cpp:1406]: I know it was not in the original code, but while you are at it, for safety, I would insert a `break;`here.

File: xbmc/video/Teletext.cpp [hunk 3]
```diff
@@ -1369,35 +1373,55 @@ void CTeletextDecoder::DoFlashing(int startrow)
...
```
Comment (ksooo) [inline xbmc/video/Teletext.cpp:1424]: I know it was not in the original code, but while you are at it, for safety, I would insert a `break;`here.

**Review by ksooo — APPROVED** [review]
Looks good.

File: xbmc/video/Teletext.cpp (hunk 2 above)
Comment (enen92) [reply xbmc/video/Teletext.cpp:1406]: added on separate commit

File: xbmc/video/Teletext.cpp (hunk 3 above)
Comment (enen92) [reply xbmc/video/Teletext.cpp:1425]: added on separate commit

**Review by ksooo — APPROVED** [review]

**Pull request merged**